roles/fio-distributed/templates/client.yaml

```diff
@@ -0,0 +1,36 @@
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: '{{ meta.name }}-fio-client'
+  namespace: '{{ operator_namespace }}'
+spec:
+  ttlSecondsAfterFinished: 600
+  template:
+    metadata:
+      labels:
+        app: fio-client
```

Comment:
Replace this with `fiod-client` would ensure no conflict takes place even if a user runs both together somehow

Reply:
Highly unlikely but alright.